docker-portainer-traefik-windows-vm/createUiDefinition.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
see pipeline results for changes that are needed here - also, you need to output the control values so they are used in deployment.

Reply:
sorry, I didn't see the failures